Gui/opensim/build.xml

```diff
@@ -252,7 +252,12 @@
             <!-- '-f': overwrite if files already exist. -->
             <arg line="-f dist/OpenSim.app '${mac.dist.outer.dir}/${mac.final.app.name}'" />
         </exec>
-
+        <!-- copy shell script to install command-line tools to top level folder -->
+        <exec executable="mv">
+            <!-- '-f': overwrite if files already exist. -->
+            <arg line="-f '${mac.dist.outer.dir}/bin/opensim-install-command-line.sh' 
+                       '${mac.dist.outer.dir}/bin/opensim-install-command-line.sh'" />
```

Comment:
This moves the file onto itself. I think the first argument should be `../${mac.final.app.name}/Contents/Resources/OpenSim/bin/opensim-install-command-line.sh`.

Reply:
@chrisdembia I tried that and it didn't move the file either, tried the moveFile Ant command and that also failed, so I reverted to your earlier version that creates a symbolic link, we can improve later as needed. Please let me know what you think.

Reply:
I would prefer copying rather than moving the file, but I think a symlink is fine and we can proceed with that until a user runs into an issue.